configuration/RealTimeTranslation/webResourceForCCaaS.js

```diff
@@ -0,0 +1,714 @@
+		if (translateToC1orC2 == Microsoft.Omnichannel.TranslationFramework.TranslateTo.C1) {
+			destLang = C1WebResourceNamespace.dictForAllConversation[conversationId]['C1Lang']; 
+		}
+		// While translating for C2 as we know the C1 language so source langauge is c1's language.
+		if (translateToC1orC2 == Microsoft.Omnichannel.TranslationFramework.TranslateTo.C2) {
+			sourceLang = C1WebResourceNamespace.dictForAllConversation[conversationId]['C1Lang'];
+			destLang = C1WebResourceNamespace.dictForAllConversation[conversationId]['finalC2Lang']; //it can still remain null. When we have no idea about C2 language
+		}
+		consoleLogHelper(conversationId, "Determined source and dest language", {
+			sourceLang,
+			destLang
+		})
+		var response = null;
+		//decides which transalation engine's api to call based on config set in initializeNewConversationInWebResource method
+		if (C1WebResourceNamespace.dictForAllConversation[conversationId]['engine'] == 'azure')
+			response = C1WebResourceNamespace.translateMessageInternalAzure(conversationId, message["text"], message['sender'], sourceLang, destLang);
+		else
+			response = C1WebResourceNamespace.translateMessageInternalGoogle(conversationId, message["text"], message['sender'], sourceLang, destLang);
+		response.then((value) => {
+			consoleLogHelper(conversationId, "Translation result received", {
+				value
+			});
+			if (value.sourceLanguage == value.destinationLanguage) {
+				value.sourceLanguage = C1WebResourceNamespace.dictForAllConversation[conversationId]['C1LangLocaleCode'];//replacing the current code with original C1 language code which came during initialization of new conversation
+				value.destinationLanguage = C1WebResourceNamespace.dictForAllConversation[conversationId]['C1LangLocaleCode'];//replacing the current code with original C1 language code which came during initialization of new conversation
+			} else if (translateToC1orC2 == Microsoft.Omnichannel.TranslationFramework.TranslateTo.C1) {
+				value.destinationLanguage = C1WebResourceNamespace.dictForAllConversation[conversationId]['C1LangLocaleCode'];//replacing the current code with original C1 language code which came during initialization of new conversation
+			} else if (translateToC1orC2 == Microsoft.Omnichannel.TranslationFramework.TranslateTo.C2) {
+				value.sourceLanguage = C1WebResourceNamespace.dictForAllConversation[conversationId]['C1LangLocaleCode'];//replacing the current code with original C1 language code which came during initialization of new conversation
+			}
+			consoleLogHelper(conversationId, "Dumpping the latest dictionary and value", {
+				dict: C1WebResourceNamespace.dictForAllConversation[conversationId],
+				value
+			});
+			return value;
+		});
+		return response;
+	},
+
+	updateC2Language: function (conversationId, detectedLang) {
+		var currentC2Language = C1WebResourceNamespace.dictForAllConversation[conversationId]?.['finalC2Lang'];

```

Comment:
I see `var` keyword is used for variables declaration, we can avoid variable declaration using `var` as much as possible. Use `let` and `const` wherever applicable.